bsip-00XX Dynamic market fees.md

```diff
@@ -0,0 +1,183 @@
+    BSIP: TBD
+    Title: Dynamic Market Fees
+    Authors: OpenLedgerApp <https://github.com/OpenLedgerApp>
+    Status: Draft
+    Type: Protocol
+    Created: 2018-12-17
+    Updated: 2018-12-17
+    Discussion: 
+    Worker: TBD
+    
+# Abstract
+UIA owners do not have many marketing tools to increase trade volume of their assets.
+
+One of the ways to increase the trade volume can be Dynamic Market Fees that will provide additional discounts for active traders.
+For example, to set lower fees for trading larger volumes.
+
+So, instead of market fee, an asset owner can optionally set up dynamic_market_fee_percent property that may look like a following example table.
+**Please refer to the general notes below for terms (maker, taker) explanation.**
+
+Trade daily volume | Maker fee, % | Taker fee, %
+---|---|---
+<=500 | 0.19% | 0.20%
+\> 500 | 0.18% | 0.20%                        
+\> 1K | 0.17% | 0.20%                        
+\> 2.5K | 0.16% | 0.19%                        
+\> 5K | 0.15% | 0.18%                        
+\> 7.5K | 0.14% | 0.17%                        
+\> 10K | 0.1% | 0.15%                        
+\> 15K | 0.075% | 0.13%                        
+\> 20K | 0.05% | 0.12%                        
+\> 25K | 0.025% | 0.11%                        
+\> 50K | 0.01% | 0.07%                        
+
+
+## Motivation
+- To provide additional marketing tools to encourage traders. 
+- To increase trading volume on BitShares. 
+- To attract traders to the ecosystem.
+
+# Rationale
+
+## Use cases:
+**1) Supporting market makers.**
+Every trade occurs between two parties: the maker, whose order exists on the order book prior to the trade, and the taker, who places the order that matches (or "takes") the maker's order.
+
+Makers are so named because their orders make the liquidity in a market. Takers are the ones who remove this liquidity by matching makers' orders with their own.
+
+The maker-taker model encourages market liquidity by rewarding the makers of that liquidity with a fee discount. It also results in a tighter market spread due to the increased incentive for makers to outbid each other. The higher fee that the taker pays is usually offset by the better prices this tighter spread provides.
+
+**2) Supporting heavy volume traders.**
+Fees are charged and deduced on a per-trade basis. The bigger “trading volume” users have traded, the lower fee they get on subsequent trades.
+
+# General notes
+Each asset will have either usual market fee or a table with dynamic market fees that will represent the table above. 
+
+A switch **market_fee_type (REGULAR, DYNAMIC)** should clearly indicate, which market fee type is used.
+
+**Makers** are the ones who put up their orders first.
+
+And **takers** are the ones to put up their orders later in time than makers. 
```

Comment:
I's suggest to enhance this BSIP by allowing maker/taker differentiation also for normal market fee (default is only one value for market fee, which is then used for maker and taker)

Reply:
It's a good point! We will update BSIP.